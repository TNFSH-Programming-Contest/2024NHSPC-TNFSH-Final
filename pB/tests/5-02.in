100 10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
250 2967 157985 256 66692597 8607 39 27729879 144 590170
0 
3 5 9 6
26
2 76 3 90 7
1 10
19
3 32 3 44 6 96 3
10 8 9 6 5 3 7 4 10 2 1
33
6 18 6 17 5 46 6 4 6 90 3 66 8
8 9 10 3 2 7 6 1 4
31
10 79 6 52 1 49 8 33 3 6 3 70 3 20 1 12 8 54 8 68 5
9 6 8 7 2 10 1 5 9 3
24
8 15 8 72 8 11 3 5 9 22 9 46 9 100 2 31 4
4 1 6 2 10
33
9 83 3 17 2 75 10 99 8 34 4 71 2 40 1 47 4 32 10
0 
65
2 46 10 3 9
9 2 10 8 1 9 3 4 6 7
71
3 20 9 56 9 64 10
2 4 5
75
5 66 6 50 2 27 1 48 7 45 9
4 3 9 5 6
77
0 
5 1 7 4 2 10
20
10 76 4 55 4 99 10 92 3 60 3 32 10 48 8 20 10 61 3 77 6
0 
46
6 14 10 40 9 12 8 67 2 60 7 28 3
1 10
19
3 71 9 66 7 15 10
2 3 1
40
1 49 9
7 6 3 1 4 10 5 9
60
10 99 9 92 9 44 1 91 4 16 6 33 6 23 5 42 8 59 1 35 1
10 10 5 3 9 2 6 1 4 8 7
89
4 31 10 5 1 43 2 58 2
9 10 7 5 4 9 2 3 8 1
8
6 3 2 92 10 29 8 34 8 35 9 54 4
9 2 4 3 8 10 9 1 5 6
98
8 86 7 35 4 78 6 49 10 98 8 24 3 4 1 97 6
10 6 8 5 7 1 4 2 9 3 10
47
3 96 10 15 8 19 8
10 10 8 7 9 6 5 3 2 4 1
99
0 
6 6 3 2 7 9 4
69
1 52 9
8 6 1 8 7 5 4 10 3
38
4 98 9 51 6 68 3 44 6
3 3 6 1
82
0 
2 5 9
79
1 98 5
6 1 6 8 7 5 9
90
5 18 5 1 8 67 4 11 7 98 9
6 1 5 3 9 4 10
49
9 42 1 73 7 27 2 66 8 69 8 25 5 8 2 20 3 28 2
5 8 10 7 9 1
93
3 85 10 61 3 53 1
1 1
80
5 32 4 85 6 30 10 99 1 93 1
9 6 9 1 4 7 3 5 2 10
30
0 
10 7 2 6 5 1 3 9 10 8 4
52
0 
1 1
58
0 
10 4 6 10 3 1 7 8 5 9 2
75
5 40 9 14 1 15 9 47 10 46 4
10 3 6 7 9 2 5 8 10 4 1
89
9 93 4 83 6 90 2 29 7 56 3 4 5 54 5 68 8 31 4
1 10
58
1 12 8
10 2 7 1 6 8 5 10 9 4 3
59
0 
7 5 4 7 6 2 9 10
98
9 86 8 74 5 76 5 98 9 16 8 43 9 72 7 37 4 30 6
9 4 6 9 3 8 1 10 2 7
56
10 41 6 10 2 55 2 78 5 61 7 77 6 6 9 87 9 100 5 51 5
10 9 1 2 8 10 4 7 6 5 3
67
6 81 6 65 4 42 1 88 1 48 10 35 9
7 2 6 5 10 4 8 9
100
4 10 1 59 4 56 3 42 8
7 8 6 4 5 3 7 1
6
0 
2 4 9
66
4 11 3 22 2 45 4 82 8
0 
94
2 53 5 55 1
8 2 7 1 4 3 6 8 5
80
9 41 6 97 7 9 5 83 9 67 2 58 7 69 6 30 8 37 7
9 10 2 5 9 1 4 3 8 7
28
6 82 3 26 9 71 1 15 10 43 1 18 5
10 2 7 4 3 9 10 1 6 8 5
80
7 67 8 35 8 95 1 62 3 85 8 52 2 33 6
0 
31
2 14 3 17 9
4 1 5 3 2
12
1 64 1
9 5 4 6 9 8 7 3 1 10
55
1 10 2
8 5 3 6 9 7 8 1 4
22
5 12 5 18 5 97 2 42 4 6 10
5 4 1 3 5 8
57
0 
8 4 9 5 2 8 7 3 1
32
3 20 9 6 3 24 9
7 10 6 7 4 9 2 8
36
3 49 7 31 4 23 7
10 9 1 2 10 5 3 6 4 7 8
32
1 19 4
9 8 7 1 4 6 3 10 5 9
76
8 47 2 69 10 66 3 31 7 72 3 54 5 86 5 70 4
0 
64
5 4 7 65 10 46 9 1 3 61 8
7 6 4 2 5 7 8 9
31
5 87 8 10 2 100 9 54 8 45 5
5 2 4 8 3 9
28
9 31 3 66 10 14 4 89 7 100 3 2 6 58 2 17 3 72 10
10 4 7 1 8 5 3 10 6 9 2
55
10 76 7 100 9 92 5 72 4 52 10 28 4 94 4 49 10 93 1 67 2
10 3 10 7 9 1 4 2 8 5 6
30
6 12 9 18 3 27 10 40 8 91 8 52 7
2 6 8
54
0 
5 9 4 8 10 2
97
8 36 4 100 2 84 5 13 2 77 9 81 6 21 2 48 10
7 2 1 3 5 10 6 8
85
10 2 9 62 7 87 6 13 7 86 4 43 3 4 1 82 3 80 1 51 5
5 3 7 2 6 10
95
1 46 10
4 1 8 9 5
0
9 64 6 76 9 72 2 16 4 5 2 82 1 27 1 92 4 10 3
5 10 2 5 4 3
3
9 4 5 81 10 10 7 11 4 62 8 36 7 54 7 93 3 17 4
6 6 3 4 8 5 9
8
0 
10 6 4 2 1 7 8 10 3 5 9
30
0 
1 7
27
5 31 5 66 2 33 2 26 4 6 6
0 
25
6 54 5 66 10 9 2 50 6 7 4 52 6
10 1 10 6 2 3 7 8 4 9 5
82
0 
10 6 4 7 1 10 5 9 3 8 2
1
3 37 6 73 1 23 10
10 4 6 8 7 10 5 2 3 9 1
39
4 1 3 97 2 66 10 8 7
5 4 5 3 10 6
21
9 42 4 3 2 61 1 13 8 35 4 84 1 27 6 7 6 38 7
7 5 10 7 9 4 8 3
9
0 
2 9 7
75
4 61 9 97 7 86 1 76 4
10 3 9 4 6 7 10 1 2 8 5
70
6 24 5 10 8 51 3 62 9 27 10 68 7
2 8 6
24
2 5 1 90 6
10 2 8 3 1 9 5 7 4 10 6
75
8 38 2 10 5 20 3 84 2 7 2 65 9 71 7 82 6
3 10 5 6
28
5 78 9 28 9 16 6 12 6 75 7
7 8 1 3 2 7 10 4
13
0 
10 9 1 6 10 5 7 8 4 2 3
74
4 10 5 52 5 62 2 16 3
7 9 5 3 10 6 1 2
8
7 45 10 48 9 31 10 16 3 33 7 52 4 66 9
6 8 9 2 5 7 1
19
1 98 5
1 3
15
10 56 1 14 8 71 9 72 8 17 4 15 8 94 4 1 10 7 1 5 3
8 5 8 1 2 10 4 6 9
9
6 77 9 59 7 33 8 40 6 64 3 80 6
3 5 2 9
53
7 83 5 24 6 98 9 80 2 50 10 48 2 62 5
5 2 1 9 4 10
53
10 71 1 32 4 96 6 60 10 97 8 10 6 46 7 33 1 92 1 94 6
5 4 10 6 8 2
77
4 32 1 28 9 16 1 75 2
4 2 3 1 10
22
7 19 3 58 2 38 1 91 2 35 2 85 7 56 2
0 
28
0 
1 8
30
2 43 7 49 8
8 1 4 7 6 8 9 2 5
52
10 50 5 71 4 75 9 61 7 26 10 32 10 96 10 51 8 37 6 35 9
1 3
35
4 82 5 74 1 14 6 61 10
4 9 10 3 6
19
3 35 5 74 3 2 10
9 4 5 8 1 9 2 10 6 3
9
2 66 9 47 8
10 4 3 6 1 7 10 2 5 9 8
85
6 75 3 100 4 47 10 65 2 66 2 76 8
6 4 10 6 8 5 9
33
5 24 5 64 1 88 9 94 10 10 7
8 8 6 10 9 3 7 1 4
77
1 51 7
10 10 8 3 9 7 2 1 5 6 4
48
9 77 3 61 7 1 6 59 5 21 7 70 1 71 4 20 9 31 7
3 4 2 6
1
