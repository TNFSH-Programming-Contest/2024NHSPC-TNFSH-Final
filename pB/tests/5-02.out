66701374
-1
-1
-1
-1
-1
-1
-1
-1
-1
593702
-1
-1
-1
-1
-1
-1
-1
-1
-1
170067
-1
-1
66692820
-1
-1
-1
-1
-1
95182946
308
95182969
-1
-1
-1
67294878
-1
-1
-1
-1
466
-1
-1
-1
-1
-1
-1
-1
-1
-1
94584149
-1
-1
-1
-1
-1
-1
-1
-1
-1
28323513
-1
-1
-1
-1
-1
95182924
66
-1
-1
95182895
-1
-1
-1
258
-1
-1
-1
-1
-1
95182968
-1
-1
-1
-1
-1
-1
-1
-1
-1
27729909
-1
-1
-1
-1
-1
-1
-1
95183166
-1
